vdirsyncer/storage/singlefile.py

```diff
@@ -40,14 +41,33 @@ class SingleFileStorage(Storage):
         This storage is very slow, and that is unlikely to change. You should
         consider using :storage:`filesystem` if it fits your usecase.
 
-    :param path: The filepath to the file to be written to.
+    :param path: The filepath to the file to be written to. If collections are
+        used, this should contain ``%s`` as a placeholder for the collection
+        name.
     :param encoding: Which encoding the file should use. Defaults to UTF-8.
 
     Example for syncing with :storage:`caldav`::
```

Comment:
Hello @gour, could you try this out? The example for using collections with singlefile storage starts here.